### PR TITLE
Update System.IO.Packaging to 8.0.1

### LINKED ISSUE #1806: System.IO.Packaging version 8.0.0 transitive reference issue via DocumentFormat.OpenXml version 3.1.0 - CVE-2024-43483, CVE-2024-43484
**Describe the bug**
I encountered an issue with a transitive reference to System.IO.Packaging version 8.0.0 when using DocumentFormat.OpenXml version 3.1.0. The package reference appears as a warning in the NuGet package manager (screenshot attached).

**Screenshots**
Image
Image

**To Reproduce**

1. Add `DocumentFormat.OpenXml` version 3.1.0 to the project.
2. Observe the transitive dependency on `System.IO.Packaging` 8.0.0 in the package manager.

**Observed behavior**
A warning is displayed in the NuGet package manager.

**Expected behavior**
No warning or a clear explanation of the transitive dependency being safe to use.

**Desktop (please complete the following information):**
 - OS: Windows 11, Alpine 3.20.3
 - .NET Target: .NET Core 8
 - DocumentFormat.OpenXml Version: 3.1.0

### PR DESCRIPTION
**Pull request by mikeebowen**
- closes #1806

### CODE REVIEW

**Review by AlfredHellstern — APPROVED** [review]

File: Directory.Packages.props [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Project>
   <ItemGroup>
     <PackageVersion Include="BenchmarkDotNet" Version="0.13.12" />
-    <PackageVersion Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.12" />    
+    <PackageVersion Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.12" />
```
Comment (tomjebo) [inline Directory.Packages.props:5]: doesn't look like this line needed to change right?

File: Directory.Packages.props [hunk 2]
```diff
@@ -31,4 +31,4 @@
     <PackageVersion Include="xunit" Version="2.9.1" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.8.1" />
   </ItemGroup>
-</Project>
+</Project>
```
Comment (tomjebo) [inline Directory.Packages.props:34]: this one also maybe?

**Review by tomjebo — APPROVED** [review]
Otherwise, this looks good.

File: Directory.Packages.props (hunk 1 above)
Comment (mikeebowen) [reply Directory.Packages.props:5]: It's removing the extra space after the element. It happened automatically with the update.

File: Directory.Packages.props (hunk 2 above)
Comment (mikeebowen) [reply Directory.Packages.props:34]: I think this one is adding a blank line at the end of the file.

**Pull request merged**